Compile error in _cmormodule.c

Compiling the latest version of CMOR results in the following error.

---
  /usr/local/miniforge3/envs/cmor_dev/bin/x86_64-conda-linux-gnu-cc -I/usr/local/miniforge3/envs/cmor_dev/include -I./include -I./include/cdTime -DNDEBUG -D_FORTIFY_SOURCE=2 -O2 -isystem /usr/local/miniforge3/envs/cmor_dev/include -fPIC -DCOLOREDOUTPUT -I/usr/local/miniforge3/envs/cmor_dev/lib/python3.12/site-packages/numpy/_core/include -Iinclude -Iinclude/cdTime -I/usr/local/miniforge3/envs/cmor_dev/include -I/usr/local/miniforge3/envs/cmor_dev/include -I/usr/local/miniforge3/envs/cmor_dev/include -I/usr/local/miniforge3/envs/cmor_dev/include -I/usr/local/miniforge3/envs/cmor_dev/include -I/usr/local/miniforge3/envs/cmor_dev/include/python3.12 -c Src/_cmormodule.c -o build/temp.linux-x86_64-cpython-312/Src/_cmormodule.o -DgFortran
  Src/_cmormodule.c: In function 'PyCMOR_set_variable_attribute':
  Src/_cmormodule.c:221:15: warning: assignment discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
    221 |         value = PyUnicode_AsUTF8(oValue);
        |               ^
  Src/_cmormodule.c: In function 'PyCMOR_set_crs':
  Src/_cmormodule.c:939:25: error: passing argument 10 of 'cmor_set_crs' from incompatible pointer type [-Wincompatible-pointer-types]
    939 |                     tn, text_nms, CMOR_MAX_STRING,
        |                         ^~~~~~~~
        |                         |
        |                         char (*)[1024]
  In file included from ./include/cmor.h:664,
                   from Src/_cmormodule.c:6:
  ./include/cmor_func_def.h:280:43: note: expected 'char *' but argument is of type 'char (*)[1024]'
    280 |                     int ntextparam, char *text_attributes_names, int ltextparams,
        |                                     ~~~~~~^~~~~~~~~~~~~~~~~~~~~
  error: command '/usr/local/miniforge3/envs/cmor_dev/bin/x86_64-conda-linux-gnu-cc' failed with exit code 1
  [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for CMOR
  Running setup.py clean for CMOR
Failed to build CMOR
ERROR: Failed to build installable wheels for some pyproject.toml based projects (CMOR)
make: *** [Makefile:154: python] Error 1
---

I would like to know how to handle it.

--- a/Src/_cmormodule.c
+++ b/Src/_cmormodule.c
@@ -936,7 +936,7 @@ static PyObject *PyCMOR_set_crs(PyObject * self, PyObject * args)
       cmor_set_crs(gid, name, n, (char *)nms,
                     CMOR_MAX_STRING, param_val,
                     (char *)units, CMOR_MAX_STRING,
-                    tn, text_nms, CMOR_MAX_STRING,
+                    tn, (char *)text_nms, CMOR_MAX_STRING,
                     text_vals, text_len);
 
     free(text_vals);

